exercises/every/every.js

```diff
@@ -13,11 +13,32 @@
  */
 function every(collection, predicate) {
   // This is your job. :)
+  let total  = 0;
+  for (item of collection) {
+    if (predicate(item)) {
+      total += 1;
```

Comment:
You can return as soon as the predicate is `false`, like so:

```javascript
function every(collection, predicate) {
  for (let item of collection) {
    if (!predicate(item)) {
      return false;
    }
  }

  return true;
}
```

Reply:
But also, since you've already written `count`, you could do the following to implement the same logic as your original code:

```javascript
function every(collection, predicate) {
  return count(collection, predicate) === 0;
}
```